src/pages/leadTable.ts

```diff
@@ -1,31 +1,30 @@
 import template from '../pages/leadTable.html';
 import { Grid } from 'gridjs';
-
+import axios from 'axios';
+interface ILead {
+  createDate:string
+  title:string
+  contactName:string
+  email:string
+  phone:string
+  city:string
+  country:string
+  language:string
+}
 class LeadTable {
   response = {};
   constructor() {
     this.response = new Grid({
       columns: ['Creation Date', 'Title', 'Name', 'Email', 'Phone', 'City', 'Country', 'Language'],
       server: {
-        url: 'https://api.scryfall.com/cards/search?q=Inspiring',
-        data: (opts) => {
-          return new Promise((resolve, reject) => {
-            const xhttp = new XMLHttpRequest();
-            xhttp.onreadystatechange = function () {
-              if (this.readyState === 4) {
-                if (this.status === 200) {
-                  const resp = JSON.parse(this.response);
-                  resolve({
-                    data: resp.data.map(lead => [lead.released_at, lead.rarity, lead.name, lead.artist + '@mail.com', lead.collector_number + '-333', 'La Paz', lead.artist, lead.lang]),
-                    total: resp.total_cards,
-                  });
-                } else {
-                  reject();
-                }
-              }
-            };
-            xhttp.open("GET", opts.url, true);
-            xhttp.send();
+        data: () => {
+          return new Promise((resolve) => {
+            axios.get('http://localhost:3000/leads').then(({ data }) => {
```

Comment:
using localhost limits the operation of the project, if the endpoint is running on a local server.
We could make use of environment variables.  https://webpack.js.org/guides/environment-variables/